content/en/docs/install-operate/source.md

```diff
@@ -453,36 +428,6 @@ When enabling this you will be able to make the `bpf` target after:
 make bpf
 ```
 
-### Build using falco-builder container
```

Comment:
This is also removed because it's out of date. If we want to set up a new build container we can document it better, but since the one mentioned here is no longer maintained I would suggest to remove this